tensorboard/webapp/customization/customizable_component.ts

```diff
@@ -58,6 +58,7 @@ import {
  *
  *    @Injectable()
  *    @Component({
+ standalone: false,
```

Comment:
This comment shouldn't need to updated :slightly_smiling_face: 
```suggestion
```

Reply:
Ah, my search-and-replace was _almost_ perfect. Fixed.